include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -274,6 +276,7 @@ __radix_sort_scan_submit(_ExecutionPolicy&& __exec, ::std::size_t __scan_wg_size
         __hdl.depends_on(__dependency_event);
         // access the counters for all work groups
         oneapi::dpl::__ranges::__require_access(__hdl, __count_rng);
+        sycl::accessor __no_op_acc(__no_op_flag, __hdl, sycl::write_only);
```

Comment:
I think you should be able to avoid a copy to the device of this flag by adding `sycl::no_init`, and adjusting the if statement in the kernel to have an `else` which assigns it to false (but you'll have to separate the selection of the work item and the real conditional).

Reply:
That code and comment is not actual any more.

Reply:
yep, thanks.